src/crt/sllcmpu.src

```diff
@@ -0,0 +1,70 @@
+; Performs 48 bit ("short long long") unsigned comparison
+; 
+; Arguments:
+; uiy = Most significant bytes of arg 1
+; ubc = Least significant bytes of arg 1
+; ude = Most signficant bytes of arg 2
+; uhl = Least significant bytes of arg 2
+; 
+; Returns:
+; flags updated according to comparison
+
+	assume	adl=1
+
+	section	.text
+	public	__sllcmpu
+__sllcmpu:
+	push ix
+	ld ix, 0
+	add ix, sp
+
+	ld hl, (ix+18) ;hl = iy
```

Comment:
Are the parameters passed on the stack for this function? If so, update the documentation

Reply:
Values aren't passed onto the stack during the routine, but to be slightly faster, they are taken off the stack because I saw llcmpu do something similar. Documentation is updated.

Reply:
llcmpu could probably use some documentation too, but it seems to be passing one 64-bit parameter in registers and the other on the stack (since there aren't enough registers for both).